src/EventListener/LogRevisionsListener.php

```diff
@@ -523,7 +536,11 @@ private function saveRevisionEntityData(EntityManagerInterface $em, ClassMetadat
         $conn = $em->getConnection();
 
         $params = [$this->getRevisionId($conn), $revType];
-        $types = [\PDO::PARAM_INT, \PDO::PARAM_STR];
+        if (\version_compare(InstalledVersions::getVersion('doctrine/dbal') ?? '', '4.0.0', '<')) {
```

Comment:
Same here (Maybe checking for class_exists(ParameterType::class) ? I dunno if it was introduced in 4.0)